CenterForOpenScience/ember-osf-web
# [ENG-2275] Add warning for multiple users edit

- Ticket: [ENG-2275] 
- Feature flag: n/a

## Purpose
- Add warning for multiple users editing a draft simultaneously

## Summary of Changes
- Add new section for multiple user warning to drafts.draft template's right-sidebar
- some styling changes
![Screen Shot 2021-01-22 at 12 23 55 PM](https://user-images.githubusercontent.com/51409893/105523700-b7a36080-5cac-11eb-8f39-e8fc266c6fc4.png)

## Side Effects
- For small vertical screen sizes, if a user scrolls all the way down, the warning text and the navigation/register buttons would overlap, so I have decided to just have the navigation/register buttons "go over" the multi-user warning. If there is better a way handling this, please let me know!
![Screen Shot 2021-01-22 at 12 14 58 PM](https://user-images.githubusercontent.com/51409893/105523375-48c60780-5cac-11eb-92a4-a60f2b81f613.png)


## QA Notes
- Please check screen sizes and browsers for behavior. This message should be shown for all pages in the draft.

[ENG-2275]: https://openscience.atlassian.net/browse/ENG-2275
